modules/ducktests/tests/checks/utils/check_controls_util_parameters.py

```diff
@@ -0,0 +1,203 @@
+from ignitetest.services.utils.control_utility import ControlUtility
+
+DEFAULT_ADMIN_KEYSTORE = 'admin.jks'
+INSTALL_ROOT = '/opt'
+CERTIFICATE_DIR = '/opt/ignite-dev/modules/ducktests/tests/certs/'
+
+
+class Cluster:
+    """
+    Need this to initialise ControlUtility
+    """
+
+    def __init__(self, globals_dict):
+        self.context = Context(globals_dict)
+        self.instal_root = CERTIFICATE_DIR
+
+
+class Context:
+    """
+    Need this to initialise ControlUtility
+    """
+
+    def __init__(self, globals_dict):
+        self.logger = ''
+        self.globals = globals_dict
+        self.globals['install_root'] = INSTALL_ROOT
+
+
+def compare_ssl(class1, class2):
+    """
+    Compare two SslContext objects
+    """
+
+    if class1 is None and class2 is None:
+        return True
+    if isinstance(class1, SslContext) and isinstance(class2, SslContext):
+        return class1.__dict__ == class2.__dict__
+    return False
+
+
+class TestParams:
+    """
+    Globals and Kwargs for tests
+    """
+
+    test_ssl_jks = {'key_store_jks': 'admin1.jks', 'key_store_password': 'qwe123',
+                    'trust_store_jks': 'truststore.jks', 'trust_store_password': 'qwe123'}
+    test_ssl_path = {'key_store_path': '/opt/certs/admin1.jks', 'key_store_password': 'qwe123',
+                     'trust_store_path': '/opt/certs/truststore.jks',
+                     'trust_store_password': 'qwe123'}
+
+    test_ssl_context_jks = SslContext(root_dir=INSTALL_ROOT,
+                                      key_store_jks='admin1.jks', key_store_password='qwe123',
+                                      trust_store_jks='truststore.jks', trust_store_password='qwe123')
+    test_ssl_context_path = SslContext(key_store_path='/opt/certs/admin1.jks', key_store_password='qwe123',
+                                       trust_store_path='/opt/certs/truststore.jks', trust_store_password='qwe123')
+    test_ssl_context = SslContext(root_dir=INSTALL_ROOT, key_store_jks='admin1.jks')
+    test_ssl_context_default = SslContext(root_dir=INSTALL_ROOT,
+                                          key_store_jks='admin.jks', key_store_password='123456',
+                                          trust_store_jks='truststore.jks', trust_store_password='123456')
+
+    test_username = 'admin1'
+    test_password = 'qwe123'
+    test_username2 = 'admin1'
+
+
+class CheckCaseGlobalsSsl:
+    """
+    Check that control_utulity.py correctly parse SSL params from globals
+    """
+
+    @staticmethod
+    @pytest.mark.parametrize('test_globals, expected',
+                             [({'use_ssl': True,
+                                'admin': {'ssl': TestParams.test_ssl_jks}}, TestParams.test_ssl_context_jks),
+                              ({'use_ssl': True,
+                                'admin': {'ssl': TestParams.test_ssl_path}}, TestParams.test_ssl_context_path),
+                              ({'use_ssl': True}, TestParams.test_ssl_context_default)])
```

Comment:
Could we make "test_globals, expected" pair more readable? eg. test_globals at the first line and expected at the second where newline required.